SiT.Scheduler.Data.PostgreSql/DbContextFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SiT.Scheduler.Data.PostgreSql
+{
+    class DbContextFactory : IDesignTimeDbContextFactory<PostgreDbContext>
+    {
+        public PostgreDbContext CreateDbContext(string[] args)
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<PostgreDbContext>();
+            optionsBuilder.UseSqlServer(Console.ReadLine());
```

Comment:
This will connect to SqlServer. And we want to use PostgreSql.

Reply:
You cannot use `Console.ReadLine()` here. Passing it throughout the command line means to use the `args` parameter in order to read the requested information.

Reply:
I used .UseNpgsql method instead of .UseSqlServer hope it is the right one now.